mooncake-store/include/master_service.h

```diff
@@ -168,6 +171,12 @@ class MasterService {
     std::shared_ptr<BufferAllocatorManager> buffer_allocator_manager_;
     std::shared_ptr<AllocationStrategy> allocation_strategy_;
 
+#ifdef USE_LRU_MASTER
+    // LRU statistics
+    std::list <std::string> all_key_list_;
```

Comment:
Use a class (e.g., class LRUList {....}) to wrap these two LRU lists.

Reply:
yep, and we'd better add some basic tests for the new class.